src/main/java/net/dv8tion/jda/internal/entities/AbstractChannelImpl.java

```diff
@@ -166,8 +164,26 @@ public boolean isSynced()
     {
         if (getParent() == null)
             return true; // Channels without a parent category are always considered synced. Also the case for categories.
-        
-        return Helpers.deepEqualsUnordered(getParent().getPermissionOverrides(), getPermissionOverrides());
+
+        Stream<PermissionOverride> thisStream = getRolePermissionOverrides().stream().filter(
+                po -> po.getAllowedRaw() != 0 || po.getDeniedRaw() != 0
+        );
+        Stream<PermissionOverride> parentStream = getParent().getRolePermissionOverrides().stream().filter(
+                po -> po.getAllowedRaw() != 0 || po.getDeniedRaw() != 0
```

Comment:
The client also says its not synced for empty overrides

Reply:
In my testing, in the JDA there was an empty one in the category and it was not present in the channel. My client showed it as synced. That is why I filtered them out